docker/Dockerfile-ca

```diff
@@ -2,15 +2,34 @@
 # between this file and Dockerfile-kssh.
 FROM ubuntu:18.04
 
+# Dependencies
 RUN apt-get -qq update
-RUN apt-get -qq  install curl software-properties-common -y
+RUN apt-get -qq  install curl software-properties-common ca-certificates gnupg -y
 RUN useradd -ms /bin/bash keybase
 USER keybase
 WORKDIR /home/keybase
+
+# Download and verify the deb
+# Key fingerprint from https://keybase.io/docs/server_security/our_code_signing_key
 RUN curl --remote-name https://prerelease.keybase.io/keybase_amd64.deb
-USER root
+RUN curl --remote-name https://prerelease.keybase.io/keybase_amd64.deb.sig
+# GPG has no easy way of getting the ID associated with a key, so we do this :(
+# This line will error if the fingerprint of the key in the file does not match
+# Key fingerprint from https://keybase.io/docs/server_security/our_code_signing_key
+# Note that we do it this way rather than via the key servers since pulling from the
+# key servers caused a flakey build
+RUN gpg --with-colons --fingerprint $( \
+        curl -sSL https://keybase.io/docs/server_security/code_signing_key.asc | \
+        gpg --import 2>&1 | \
+        grep -v created | \
+        head -n 1 | \
+        cut -d ' ' -f 3 | \
+        cut -d ':' -f 1 \
+    ) | grep fpr | cut -d ':' -f 10 | grep 222B85B0F90BE2D24CFEB93F47484E50656D16C7
```

Comment:
can this be done in two steps instead of one? like, import the key and then run a check that it's definitely loaded? 
`gpg --fingerprint 222B85B0F90BE2D24CFEB93F47484E50656D16C7`
perhaps that will prevent needing to cut around colons.

Reply:
Good point! Updated to avoid all of the nasty bash-fu.